### ISSUE TITLE
'selected' attr doesn't seem work

### ISSUE THREAD

**Issue by alexhung** [cibernox/ember-power-select-with-create#3]
When I use `ember-power-select` with `selected=someAttrs` the attributes displays correctly.

But when I switch to using `ember-power-select-with-create` (just changing the component name, all other attributes stay same), the selected no longer work and the attributes are not displayed any more.

Help?

**Comment by felixkiss** [comment]
It works for me. Can you share some code or even a repo?

What version are you on?

**Comment by alexhung** [comment]
I'm on ember 1.13.11, ember cli 1.13.13, ember-power-select 0.7.0, and ember-power-select-with-create 0.1.2.

**Comment by alexhung** [comment]
This works.

```
{{#power-select options=tags selected=feature.tags onchange=(action (mut feature.tags)) oncreate=(action "createTag") multiple=true as |tag|}}
    {{tag.name}}
{{/power-select}}
```

This doesn't

```
{{#power-select-with-create options=tags selected=feature.tags onchange=(action (mut feature.tags)) oncreate=(action "createTag") multiple=true as |tag|}}
    {{tag.name}}
{{/power-select-with-create}}
```

`feature` is a ember data object with `tags` a `hasMany`, `async = true` attr, `tags` is an array of `Tag` ember data object.

**Comment by felixkiss** [comment]
Will be fixed in the `v0.1.3` release.

In the meantime, you can try it out by using the branch in `package.json`:
```
"ember-power-select-with-create": "cibernox/ember-power-select-with-create#feature/add-support-for-multiple"
```

**Comment by alexhung** [comment]
:+1:

**Comment by felixkiss** [comment]
Released in `0.1.3`
